pkg/credentialprovider/azure/azure_credentials.go

```diff
@@ -57,7 +57,7 @@ func init() {
 	credentialprovider.RegisterCredentialProvider("azure",
 		&credentialprovider.CachingDockerConfigProvider{
 			Provider:    NewACRProvider(flagConfigFile),
-			Lifetime:    1 * time.Minute,
+			Lifetime:    5 * time.Second,
```

Comment:
I'm a little concerned about throttling, we don't have backoff yet on token exchange. Do we know the currently allowed requests count?

Reply:
@sivagms @yuwaMSFT2 do you know what's the limit of get token from ACR(using managed identity, `receiveChallengeFromLoginServer`), we want to decrease the cache to 5s, so it's 12 per minute with this PR, related Get token from ACR code is here:
https://github.com/kubernetes/kubernetes/blob/b9f4180dca741635453b11cfec0497c996374b7c/pkg/credentialprovider/azure/azure_acr_helper.go#L78-L90

Reply:
the main concern is on `receiveChallengeFromLoginServer` using managed identity, for service principal, since it's only generating an array every 5s, there is no side effect on SP.

Reply:
/cc @sajayantony

Reply:
I don't see a problem on the frequency. But I will leave @sajayantony to confirm. Thanks!

Reply:
We are in the process of revisiting these throttles. 
mnltejaswini